Buildings/Controls/OBC/CDL/Continuous/Line.mo

```diff
@@ -62,57 +67,81 @@ equation
           lineColor={192,192,192},
           fillColor={192,192,192},
           fillPattern=FillPattern.Solid),
-        Line(points={{-86,-40},{88,48}}),
-        Line(points={{-80,-80},{-80,66}},
+        Line(points={{-46,40},{50,-44}},
+          color={0,0,0},
+          thickness=0.5),
+        Line(points={{-80,-80},{-80,72}},
                                       color={192,192,192}),
         Polygon(
-          points={{-80,88},{-88,66},{-72,66},{-80,88}},
+          points={{-80,92},{-88,70},{-72,70},{-80,92}},
           lineColor={192,192,192},
           fillColor={192,192,192},
           fillPattern=FillPattern.Solid),
         Line(points={{-88,-78},{76,-78}},
                                       color={192,192,192}),
         Line(
-          points={{-100,80},{38,70},{56,-78}},
+          points={{-100,80},{-64,80},{-62,-78}},
```

Comment:
The two-section line causes error in OMEdit (the same one as #1147). I updated them to be two lines with single section.